Makefile

```diff
@@ -125,14 +126,20 @@ GREEN := $(shell tput setaf 2)
 # We need GDAL which is hard to install in a venv, modify PYTHONPATH to use the
 # system wide version.
 GDAL_VERSION ?= 1.10.0
+PYTHON_INSTALL_VERSION ?= 3.6.4
```

Comment:
in `buildspec` file and anywhere else you are using python `3.7` maybe change the default version in the Makefile too?

Reply:
OK, but Python 3.7 won't compile locally on debian jessie